src/NServiceBus.Core/Configure.cs

```diff
@@ -126,6 +127,23 @@ internal void Initialize()
             }
         }
 
+        void ConfigureBehaviors()
```

Comment:
Even though this might be the right thing to do I'm not convinced we should do this in a patch since there might be other things that depends on the behaviors being registered at the old location. 

Have we considered other options that has a lower risk? (not saying to roll this back, just want to make sure that we are aware)